infra/e2e/turing.values.yaml

```diff
@@ -5,6 +5,14 @@ global:
 
 turing:
   uiConfig: null # Not needed as we only test API
+  # testing of openapi overriding feature
+  openApiComponentOverrides:
```

Comment:
`openApiComponentOverrides` -> `openApiSpecOverrides`. 
`openApiComponentOverrides` make it sound like you can only override `components` part of the spec file, which isn't true

Reply:
Yes I think this is a fair point, will change this.